src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/ServiceHearingsControllerTest.java

```diff
@@ -1,158 +1,158 @@
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.reform.sscs.service.AuthorisationService.SERVICE_AUTHORISATION_HEADER;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+@AutoConfigureMockMvc(addFilters = false)
+@ActiveProfiles("integration")
+class ServiceHearingsControllerTest {
+
+    private static final long CASE_ID = 1625080769409918L;
+    private static final long MISSING_CASE_ID = 99250807409918L;
+    private static final String BAD_CASE_ID = "ABCASDEF";
+    private static final String AUTHORIZATION = "Bearer eyJhbGciOiJIUzUxMiL7.eyJzdWIiOiJzc2NzIiwiZXhwIjoxNjQ2NDA5MjM5fQ"
+            + ".zSEbvMJedOGo16yBOXecLgucWyavnoVu023cterreUF0sxPlmV-Qu8Y7OloJUKrLGlNweUr8mVpYWPzE0iNyYw";
+    private static final String BAD_AUTHORIZATION = "eyJhbGciOiJIUzUxMiL7.eyJzdWIiOiJzc2NzIiwiZXhwIjoxNjQ2NDA5MjM5fQ"
+            + ".zSEbvMJedOGo16yBOXecLgucWyavnoVu023cterreUF0sxPlmV-Qu8Y7OloJUKrLGlNweUr8mVpYWPzE0iNyYw";
+    private static final String NOT_AUTHORIZATION = "Bearer notauthed";
+    private static final String CASE_REFERENCE = "caseReference";
+    private static final String SERVICE_HEARING_VALUES_URL = "/serviceHearingValues";
+    private static final String CASE_NAME = "Test Case Name";
+
+    private static final ObjectMapper mapper = new ObjectMapper().registerModule(new JavaTimeModule());
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private IdamService idamApiService;
+
+    @MockBean
+    private AuthorisationService authorisationService;
+
+    @MockBean
+    private AuthTokenGenerator authTokenGenerator;
+
+    @MockBean
+    private CcdService ccdService;
+
+    @BeforeEach
+    public void setUp() throws AuthorisationException, InvalidHeaderException {
+        SscsCaseDetails caseDetails =
+                SscsCaseDetails.builder().data(SscsCaseData.builder()
+                        .workAllocationFields(WorkAllocationFields.builder().caseNamePublic(CASE_NAME).build())
+                                .build()).build();
+        given(ccdService.getByCaseId(eq(CASE_ID), any(IdamTokens.class))).willReturn(caseDetails);
+        given(authTokenGenerator.generate()).willReturn("s2s token");
+        given(idamApiService.getIdamTokens()).willReturn(IdamTokens.builder().build());
+        willThrow(new InvalidHeaderException(new Exception("Test")))
+                .given(authorisationService).authorise(anyString());
+        willThrow(new AuthorisationException(new Exception("Test")))
+                .given(authorisationService).authorise(matches("Bearer .+"));
+        willDoNothing().given(authorisationService).authorise(AUTHORIZATION);
+    }
+
+    // These are holder tests that will need to be implemented alongside service hearing controller
```

Comment:
when will they be implemented, is there a ticket for this? or part of the same ticket?